lib/src/definitions.ts

```diff
@@ -63,6 +63,12 @@ export interface PathParam {
   name: string;
   description?: string;
   type: Type;
+  examples?: Example[];
+}
+
+export interface Example {
+  name: string;
+  value: string;
```

Comment:
OpenAPI examples will be output as strings, however within SPOT it will be useful to retain type information. For now we can leave this as `string` and leave a `TODO`.
```suggestion
  value: string; // TODO: encapsulate type information
```